src/xinspect.hpp

```diff
@@ -160,6 +160,7 @@ namespace xcpp
                     tagfile = it->at("tagfile");
                     std::string filename = tagfiles_dir + "/" + tagfile;
                     pugi::xml_document doc;
+                    pugi::xml_parse_result result __attribute__((unused)) = doc.load_file(filename.c_str());
```

Comment:
`__attribute__` is UNIX only, it should be avoided (or used through macros with different definitions depending on the platform). If you don't need `result`, you can simply call `doc.load_file` without assigning its result, or use the `(void*)result;` trick to avoid the warning.